### PR TITLE
Preparation to la v2.

### PR DESCRIPTION
**Pull request by halge**

### CODE REVIEW

File: importers/platsannons/converter.py [hunk 1]
```diff
@@ -162,6 +165,7 @@ def convert_ad(message):
     annons['timestamp'] = message.get('updatedAt')
     annons['logo_url'] = message.get('logo_url')
     # Extract labels as keywords for easier searching
+    log.info("Convert ad: %s took: %d milliseconds." % (annons['id'], int(time.time() * 1000) - start_time))
```
Comment (oskardrenske-ith) [inline importers/platsannons/converter.py:168]: I got a comment from Mattias that this logging will impact performance and fill upp the logs with messages that are not really needed

Comment (lina1101) [reply importers/platsannons/converter.py:168]: I guess we could deleted this @halge, is that ok?

Comment (lerca) [reply importers/platsannons/converter.py:168]: or change to log.debug?

Comment (halge) [reply importers/platsannons/converter.py:168]: i change it to debug.

**Pull request merged**